script.js

```diff
@@ -27,12 +30,19 @@ function endGame() {
 startButton.addEventListener('click', () => {
     splashScreen.style.display = "none"
     document.addEventListener("keydown", (event) => {
+        if(lastKeyPressed === '') {
+            raceStopwatch = setInterval(() => {
+                timer++
+                gameTimer.textContent = timer
+            }, 1000)
+        }
         if (event.code === "ArrowLeft" || event.code === "ArrowRight") {
             if (checkKey(lastKeyPressed, event.code)) {
                 lastKeyPressed = event.code
                 penguinPosition = movePenguin(penguinPosition)
                 if (checkEnd(penguinPosition)) {
                     endGame()
```

Comment:
I'd now suggest renaming the endGame function to something more semantic. 

Now that you're clearing the raceStopwatch, endGame is more like endGameMessage or something like that